libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -30,13 +30,6 @@ using namespace std;
 using namespace dev;
 using namespace dev::solidity;
 
-ABIFunctions::~ABIFunctions()
-{
-	// This throws an exception and thus might cause immediate termination, but hey,
-	// it's a failed assertion anyway :-)
-	solAssert(m_requestedFunctions.empty(), "Forgot to call ``requestedFunctions()``.");
```

Comment:
This is not really useful, it will break hard if this assert is hit. Also, it prevents regular exceptions in this module from being propagated, and finally, code that has missing functions will not compile anyway.